tests/test_multistore_file.py

```diff
@@ -27,6 +34,32 @@ def filename(self):
 
 
 class MultistoreFileTests(unittest.TestCase):
+    def tearDown(self):
+        try:
+            os.unlink(FILENAME)
+        except OSError:
+            pass
+
+    def setUp(self):
+        try:
+            os.unlink(FILENAME)
+        except OSError:
+            pass
+
+    def create_test_credentials(self, client_id='some_client_id',
```

Comment:
Since a helper, can you start the name with an underscore? I realize the previous one was not, but again, better than we found it.